sys/net/gnrc/application_layer/dhcpv6/client_6lbr.c

```diff
@@ -85,6 +86,9 @@ static void _configure_upstream_netif(gnrc_netif_t *upstream_netif)
         addr.u8[15] = 2;
         gnrc_netif_ipv6_addr_add(upstream_netif, &addr, 64, 0);
     }
+
+    /* disable router advertisements on upstream interface */
```

Comment:
Here and below: Some more in-depth reasoning would be nice. How about something like this?

```suggestion
    /* disable router advertisements on upstream interface. With this, the border
     * router
     * 1. Does not confuse the upstream router to add the border router to its 
     *    default router list and
     * 2. Solicits upstream Router Advertisements quicker to auto-configure its
     *    upstream global address.
     */
```